test/unit/directives/myHeadDirective.js

```diff
@@ -2,25 +2,23 @@
 /* global describe, it, inject, expect, beforeEach */
 
 describe('myHead directive', function () {
-  var $compile, $rootScope
-
   beforeEach(module('myApp.templates'))
   beforeEach(module('myApp.directives'))
 
   beforeEach(inject(function (_$compile_, _$rootScope_) {
-    $compile = _$compile_
-    $rootScope = _$rootScope_
+    this.$compile = _$compile_
+    this.$rootScope = _$rootScope_
   }))
 
   it('compiles a my-head attribute', function () {
-    var compiledElement = $compile('<div my-head></div>')($rootScope)
-    $rootScope.$digest()  // Fire watchers
+    var compiledElement = this.$compile('<div my-head></div>')(this.$rootScope)
+    this.$rootScope.$digest()  // Fire watchers
```

Comment:
[Number 9](https://opensource.com/article/17/5/30-best-practices-software-development-and-testing) - comments are going to lie at you at one point in time.